@utahdts/utah-design-system/react/components/forms/TimeInput.jsx

```diff
@@ -0,0 +1,171 @@
+import { add, format, isValid, parse } from 'date-fns';
+import { useMemo } from 'react';
+import { joinClassNames } from '../../util/joinClassNames';
+import { ComboBox } from './ComboBox/ComboBox';
+import { ComboBoxOption } from './ComboBox/ComboBoxOption';
+import { useFormContextInputValue } from './FormContext/useFormContextInputValue';
+import { TextInput } from './TextInput';
+
+/**
+ * @param {Date | null} date
+ * @param {Date} dateDefault
+ * @returns {Date}
+ */
+function defaultDate(date, dateDefault) {
```

Comment:
This function just sits here at the top?

Reply:
This is so small, just move it back in to the useMemo with variables.